response_operations_ui/views/case.py

```diff
@@ -110,7 +110,8 @@ def get_user_from_case_events(case_events):
         except KeyError:
             logger.info('There was no partyId in case event', case_event=case_events[first_case_event])
             return ''
-        respondent_name = respondent.get('firstName') + ' ' + respondent.get('lastName')
+        respondent_name = respondent.get('firstName') + ' ' + respondent.get('lastName') if respondent != '' \
+            else 'Respondent does not exists in the system. This may be due to the GDPR retention policy.'
```

Comment:
I think if you make the above an empty json I think you could make this

if not respondent_name:
   return Respondent does not exists in the system. This may be due to the GDPR retention policy.